src/runner/browser-env/vite/browser-modules/index.ts

```diff
@@ -1,4 +1,12 @@
+import { remote } from "webdriverio";
+import { automationProtocolPath } from "virtual:@testplane/driver";
 import { MochaWrapper } from "./mocha/index.js";
 
+window.browser = await remote({
```

Comment:
Why `window.browser`, and not `window.__testplane__.browser`?

If possible, i would move it under `__testplane__`, so user would not overwrite it

Reply:
I want the user to be able to quickly execute some command directly in devtools for debug purpose. And call command like `__testplane__.browser.getUrl()` will not be very convenient.

Reply:
Yes, this is a good solution.

> Do we use the browser from the window?

I used it inside mocha and send to runnables. But for now I use `window. __testplane__.browser` in it.